package.json

```diff
@@ -171,26 +171,27 @@
         ]
     },
     "dependencies": {
+        "ci": "^2.2.0",
```

Comment:
```suggestion
```
`ci` remove this dependency, it is not needed. Can you also run `npm install`, after removing it. It should clear up the `package-lock.json`.